### PR TITLE
ui/sidebar: change filter naming and behavior to make it clear that tests are a type of resource [ch11349]

### PR DESCRIPTION
**Pull request by maiamcc**
fyi han, the very last commit of this bunch is a shot at renaming
because lines were getting long and names were feeling clunky,
but if you object (do you have a more specific understanding of
"toggle" than I do?) lmk and i can put it back

### CODE REVIEW

**Comment by maiamcc** [comment]
Screen Recording 2021-02-15 at 6 19 19 PM

File: web/src/OverviewSidebarOptions.tsx [hunk 1]
```diff
@@ -42,6 +39,39 @@ const useStyles = makeStyles({
   },
 })
 
+const segmentedControlsBorderRadius = "3px"
```
Comment (hyu) [inline web/src/OverviewSidebarOptions.tsx:42]: Maybe make this consistent with the updated naming? `toggleBorderRadius`? Then you can replace the one remaining `border-radius: 3px;` for `AlertsOnTopToggle`.

**Review by hyu — APPROVED** [review]

File: web/src/SidebarResources.tsx [hunk 2]
```diff
@@ -120,20 +120,19 @@ export class SidebarResources extends React.Component<SidebarProps> {
...
+    let filteredItems = [...this.props.items]
+    if (options.testsHidden) {
+      filteredItems = this.props.items.filter((item) => !item.isTest)
+    } else if (options.testsOnly) {
+      filteredItems = this.props.items.filter((item) => item.isTest)
+    }
 
     if (options.alertsOnTop) {
       filteredItems.sort(sortByHasAlerts)
+    } else {
```
Comment (landism) [inline web/src/SidebarResources.tsx:135]: 👀

**Review by landism — APPROVED** [review]

File: web/src/SidebarResources.tsx (hunk 2 above)
Comment (maiamcc) [reply web/src/SidebarResources.tsx:135]: 😅 😅 😅 😅

**Pull request merged**